### PR TITLE
[DEV-4404] Acquire new source for city, county, and state data

### PR DESCRIPTION
**Pull request by kbard**
**Description:**

Our original source for city, county, and state data had been lost to the annals of time and the files backing it had grown very stale.  This pull request rectifies that by populating a new table from an established source for said data.  While the original hope was to pull the data directly from Broker, some obstacles made that difficult, so we did the next best thing and now pull from the same source as Broker.

**Requirements for PR merge:**

1. [x] Unit & integration tests updated
2. [x] API documentation should be unaffected
3. [x] Necessary PR reviewers:
    - [x] Backend
4. [x] Materialized views should be unaffected
5. [x] Front end should be unaffected
6. [x] Data validation completed
7. [x] Operations ticket created: [DEV-4457](https://federal-spending-transparency.atlassian.net/browse/DEV-4457)
8. [x] Jira Ticket [DEV-4404](https://federal-spending-transparency.atlassian.net/browse/DEV-4404):
    - [x] Link to this Pull-Request

### CODE REVIEW

File: usaspending_api/awards/management/commands/load_subawards.py [hunk 1]
```diff
@@ -49,7 +51,7 @@ def _perform_load(self):
         broker_subaward = ETLTable("broker_subaward")
         subaward = ETLTable("subaward")
 
-        remote_subaward = ETLDBLinkTable("subaward", "broker_server", broker_subaward.data_types)
+        remote_subaward = ETLDBLinkTable("subaward", settings.DATA_BROKER_DBLINK_NAME, broker_subaward.data_types)
```
Comment (tony-sappe) [inline usaspending_api/awards/management/commands/load_subawards.py]: Wonder why it took so long to turn that into a global....

File: usaspending_api/references/migrations/0039_auto_20200206_1253.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            ],
+            options={
+                'db_table': 'ref_city_county_state_code',
+            },
+        ),
+        migrations.DeleteModel(
+            name='RefCityCountyCode',
+        ),
+
+        # Special index used to guarantee uniquity on the natural key columns.
```
Comment (tony-sappe) [inline usaspending_api/references/migrations/0039_auto_20200206_1253.py]: "guarantee uniquity" how? It isn't a unique index

File: usaspending_api/references/management/commands/load_city_county_state_code.py [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+from usaspending_api.common.etl import ETLTable, ETLTemporaryTable, operations
+from usaspending_api.common.helpers.sql_helpers import execute_dml_sql
+from usaspending_api.common.helpers.timing_helpers import Timer
+from usaspending_api.common.retrieve_file_from_uri import RetrieveFileFromUri
+from usaspending_api.common.zip import extract_single_file_zip
+from usaspending_api.etl.operations.subaward.update_city_county import update_subaward_city_county
+from usaspending_api.references.models import CityCountyStateCode
+
+
+logger = logging.getLogger("console")
```
Comment (tony-sappe) [inline usaspending_api/references/management/commands/load_city_county_state_code.py]:
```suggestion
logger = logging.getLogger("script")
```

Better log format for management commands

File: usaspending_api/references/migrations/0039_auto_20200206_1253.py (hunk 2 above)
Comment (kbard) [reply usaspending_api/references/migrations/0039_auto_20200206_1253.py]: oversight

**Pull request merged**